UNRELEASED.md

```diff
@@ -17,6 +17,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Bug fixes
 
+- Fixed FooterHelp component example by deleting a period ([2152](https://github.com/Shopify/polaris-react/pull/2152))
```

Comment:
This isn't code related so it's not a bug fix, can you put it in the documentation section.

Also the convention is to include a # before . the issue number (so `#2152`)